truss/truss_handle.py

```diff
@@ -150,6 +150,51 @@ def get_docker_image(self, labels: Dict):
         """[Deprecated] Do not use."""
         return _docker_image_from_labels(labels)
 
+    @proxy_to_shadow_if_scattered
+    def run_python_script(
+        self,
+        script_path: Path,
+        build_dir: Optional[Path] = None,
+    ):
+        image = self.build_serving_docker_image(build_dir=build_dir)
+        secrets_mount_dir_path = _prepare_secrets_mount_dir()
+
+        envs: Dict[str, str] = {}
+        # Add bundled packages to the PYTHONPATH. Note
+        # that this is necessary to achieve the same environment as Truss Server
+        # -- this is setup that is done by Truss Server, that won't be available
+        # to the standalone script.
+        bundled_packages_path = Path("/packages")
+        envs["PYTHONPATH"] = bundled_packages_path.as_posix()
+
+        return Docker.client().run(
+            image.id,
+            entrypoint="python",
+            command=["/app/script.py"],
+            detach=False,
+            mounts=[
+                [
+                    "type=bind",
+                    f"src={str(secrets_mount_dir_path)}",
+                    "target=/secrets",
+                ],
+                [
+                    "type=bind",
+                    f"src={str(LocalConfigHandler.bptr_data_resolution_dir_path())}",
+                    "target=/bptr",
+                ],
```

Comment:
I'm not sure I know what's happening here. What's bptr?

Reply:
good question, I copied this from the other run block: https://github.com/basetenlabs/truss/blob/34447ceca3e0f1ccf51b475247333d1b141c1f2b/truss/local/local_config_handler.py#L73, I have no idea what bptr is. Might be some legacy thing, will remove, it seems to work without it.